code/modules/ruins/spaceruin_code/forgottenship.dm

```diff
@@ -172,7 +172,6 @@ GLOBAL_VAR_INIT(fscpassword, generate_password())
 /mob/living/simple_animal/hostile/nanotrasen/elite
 	name = "Nanotrasen Elite Assault Officer"
 	desc = "Pray for your life, syndicate. Run while you can."
-	icon = 'icons/mob/simple_human.dmi'
```

Comment:
Have you confirmed this isn't necessary?

Reply:
yes